dentapp/src/components/registerInput.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { InputGroup, FormControl } from "react-bootstrap";
+
+class RegisterInput extends React.Component {
+  state = {
+    input: "",
+  };
+
+  handleChange = (e) => {
+    this.setState({ input: e.target.value });
+    console.log("input value: ", e.target.value);
+  };
+
+  render() {
```

Comment:
The props for this component should be:
- Placeholder
- Title
- Margin color
- Font Color
- Input Type
- onHandleChange - you already have it

Instead of two components, make just one